### ISSUE TITLE
'tensorboard.plugins.projector' has no attribute 'ProjectorConfig'

### ISSUE THREAD

**Issue by marcevrard** [tensorflow/tensorboard#926]
Referring to the test file:
https://github.com/tensorflow/tensorboard/blob/1.5/tensorboard/plugins/projector/projector_api_test.py

If `projector` is imported as:
```python
from tensorboard.plugins import projector
config = projector.ProjectorConfig()
```
accessing the `ProjectorConfig()` raises:
`AttributeError: module 'tensorboard.plugins.projector' has no attribute 'ProjectorConfig'`

It works fine if `projector` is imported from `tf.contrib`:
```python
from tensorflow.contrib.tensorboard.plugins import projector
```
---
```
tensorboard == 1.5.0
tensorflow == 1.5.0
Python == 3.6.3
Ubuntu == 17.10
```

**Comment by chihuahua** [comment]
Hmm, importing `ProjectorConfig` should work due to this line:
https://github.com/tensorflow/tensorboard/blob/f6bb9b09371c2b1e2107ac5c26b205528eb0ff08/tensorboard/plugins/projector/__init__.py#L35

I wonder why not. I wonder what happens if you install tensorboard nightly (`pip install tb-nightly`) and thus run on the bleeding edge version.

**Comment by marcevrard** [comment]
Just tried and got the exact same error.

**Comment by nfelt** [comment]
Upon further investigation, it looks like the actual contents of our `__init__.py` files isn't being preserved during our pip package building.  We didn't notice because most of them are empty (aside from a license) and Bazel evidently creates its own empty `__init__.py` files by default: https://github.com/bazelbuild/bazel/pull/4470

The projector plugin is pretty much the only non-empty one aside from the top-level one, which is specifically listed in its Bazel rule and does get included.  I think if we start explicitly listing the other ones that should fix this issue.

**Comment by nfelt** [comment]
For now, a workaround should be to explicitly import from projector_config_pb2 as in

```
from tensorboard.plugins.projector.projector_config_pb2 import ProjectorConfig
config = ProjectorConfig()
```

**Comment by marcevrard** [comment]
Though the function `visualize_embeddings()` won't be available either.
My current workaround is thus to import from:
```python
from tensorflow.contrib.tensorboard.plugins import projector
```
Are there any possible issues using the `tf.contrib` module instead of the `tb` *root* module? (I guess this will be still supported for a while)

**Comment by nfelt** [comment]
Oh, yes, you're right.  I believe that will work for now but it will be safer to import from `tensorboard` once we get that working.

**Comment by marcevrard** [comment]
I checked with the latest package (`tb-nightly-1.6.0a20180206`) and it works perfectly.
Thanks for the swift fix!

**Comment by nfelt** [comment]
Sure.  This should also be fixed in 1.5.1 (just released yesterday) and will be in the upcoming 1.6.0 as well.
